### PR TITLE
Add personal website workshop

### PR DESCRIPTION
**Pull request by ad510**
This replaces #652, with some small changes so it should be ready to merge (hopefully).

### CODE REVIEW

File: workshops/personal_website/README.md [hunk 1]
```diff
@@ -0,0 +1,575 @@
...
+
+Open the [live demo][final-live-demo]. See the [final code][final-code].
+
+[final-live-demo]: https://cdn.rawgit.com/anonymous/7a5a6cc614052c8c810f/raw/9419c5a5a4a871da091a49844ffbc46f434c01c8/index.html
+[final-code]: http://jsbin.com/gist/7a5a6cc614052c8c810f?html,output
+
+**On the way, you will:**
+
+- Begin to learn the programming languages **HTML** & **CSS**
+- Setup the coding tools, **GitHub**, **Cloud9**, **Slack** which  you will use
```
Comment (zachlatta) [inline workshops/personal_website/README.md]: Misplaced comma and extra space in this line. Should be:

```
Setup the coding tools **GitHub**, **Cloud9**, and **Slack**, which you will use
```

File: workshops/personal_website/README.md [hunk 2]
```diff
@@ -0,0 +1,575 @@
...
+    to verify your account
+  - Make sure to remember your username and password, you'll need this at every
+    club meeting
+  - When prompted to choose a plan, to choose the **"Free"** plan
+2. Open your email inbox in a new tab and look for an email from GitHub. Open it
+   and click the button to verify your email
+3. Open https://gh.hackclub.io in a new tab and star the project by clicking on
+   the button that looks like this on the top right: ![](img/github_star.png)
+
+### 2) Create Your First GitHub repository
```
Comment (zachlatta) [inline workshops/personal_website/README.md]: Capitalize `repository`

File: workshops/personal_website/README.md [hunk 3]
```diff
@@ -0,0 +1,575 @@
...
+   `https://github.com/username/username.github.io.git`. You will need this link
+   for the next step.
+
+### 3) Sign Up for Cloud9
+
+Just like you can use Google Drive to write and organize documents written in
+English, we'll be using **Cloud9** to write, save, and organize our code.
+
+> The techy term for a place where you can write, and organize code is **"IDE"**
+> or "Integrated Development Environment"**
```
Comment (zachlatta) [inline workshops/personal_website/README.md]: This isn't technically true, but w/e

File: workshops/personal_website/README.md [hunk 4]
```diff
@@ -0,0 +1,575 @@
...
+3. Then name the file **`index.html`**
+4. Double click it to open it
+
+> ![](img/c9_create_index_dot_html.gif)
+
+> Notes:
+>
+> - Just like Microsoft Word Documents end in **`.doc`**, HTML files end in
+**`.html**`
+> - Your main HTML file has to be named **`index.html`**
```
Comment (zachlatta) [inline workshops/personal_website/README.md]: Should be formatted like this:

```md
>  **`.html`**
```

Comment (zachlatta) [inline workshops/personal_website/README.md]: Whoops, meant to comment on line #140.

File: workshops/personal_website/README.md [hunk 5]
```diff
@@ -0,0 +1,575 @@
...
+
+### 2) Adding standard HTML template
```
Comment (zachlatta) [inline workshops/personal_website/README.md]: Title case this line

File: workshops/personal_website/README.md [hunk 6]
```diff
@@ -0,0 +1,575 @@
...
+</html>
+```
+
+- The **`body`** of the HTML document is where all the **content** belongs. This
+  includes all of the words and images on the page.
+- The **`head`** of the HTML document is where you set the settings for the page
+  and include other code files you want on your website.
+
+
+
```
Comment (zachlatta) [inline workshops/personal_website/README.md]: Remove extra spaces

File: workshops/personal_website/README.md [hunk 7]
```diff
@@ -0,0 +1,575 @@
...
+> ![](img/html_add_name.gif)
+
+Notice that that the HTML code you write is immediately updated in the preview.
+That's why it's called a **"Live Preview"**
+
+### 6) Adding a Description
+
+1. Underneath on a new line, write one short sentence about yourself. I love my
+job as a call center representative so I'm going to write `MVP hotline
+answerer`.
```
Comment (zachlatta) [inline workshops/personal_website/README.md]: Lines 215 and 216 should be indented by two spaces

File: workshops/personal_website/README.md [hunk 8]
```diff
@@ -0,0 +1,575 @@
...
+
+```html
+<h1>Drizzy Drake</h1>
+```
+
+**HTML tags**, like this heading tag, tell the computer how to interpret the
+text in between the tags.
+
+
+
```
Comment (zachlatta) [inline workshops/personal_website/README.md]: Remove extra new lines

File: workshops/personal_website/README.md [hunk 9]
```diff
@@ -0,0 +1,575 @@
...
+1. Make your name look like a heading by surrounding your name with an `h1`
+   tag.
+
+    > Note: In Cloud9, as soon as you write  **`<h1>`**, it knows you will
+    eventually want a **`<h1>`**, it just adds it for you. This is known
+    as **autocomplete**.
+
+
```
Comment (zachlatta) [inline workshops/personal_website/README.md]: Remove extra new line

File: workshops/personal_website/README.md [hunk 10]
```diff
@@ -0,0 +1,575 @@
...
+text in between the tags.
+
+
+
+1. Make your name look like a heading by surrounding your name with an `h1`
+   tag.
+
+    > Note: In Cloud9, as soon as you write  **`<h1>`**, it knows you will
+    eventually want a **`<h1>`**, it just adds it for you. This is known
+    as **autocomplete**.
```
Comment (zachlatta) [inline workshops/personal_website/README.md]: Each line of the blockquote should have `>` preceding.

Example:

```
   > Note: In Cloud9, as soon as you write  **`<h1>`**, it knows you will
   > eventually want a **`<h1>`**, it just adds it for you. This is known
   > as **autocomplete**.
```

File: workshops/personal_website/README.md [hunk 11]
```diff
@@ -0,0 +1,575 @@
...
+  the **text** within the body tag to be **centered**
+
+This centers everything on our page because all of the content in our HTML file
+is written inside the body tag.
+
+And because everything is in the **body** tag, everything is centered.
+
+> You can Google ["CSS text align"]
+  (https://www.google.com/?gws_rd=ssl#q=css+text+align) to see what other
+  alignment options are available besides **`center`**
```
Comment (zachlatta) [inline workshops/personal_website/README.md]: Same thing as previous blockquotes. Needs to have `>` at the beginning of each line.

File: workshops/personal_website/README.md (hunk 5 above)
Comment (ad510) [inline workshops/personal_website/README.md]: In general the workshop is pretty inconsistent in terms of header capitalization. I personally don't think it's a big deal. Is this one of the things that investors kept pointing out (in addition to spelling errors)?

File: workshops/personal_website/README.md [hunk 12]
```diff
@@ -0,0 +1,575 @@
...
+5. Now try to view your website by going to `username.github.io/personal-website`
+
+   > Make sure to change `username` to your own username
+
+#### 3) Celebrate!
+
+![](img/celebrate_rush_hour.gif)
+
+Yes! Your website is now public on the internet and will stay!
+Feel free to share this with your friends!
```
Comment (zachlatta) [inline workshops/personal_website/README.md]: These two lines should be reflowed to look like this:

```
Yes! Your website is now public on the internet and will stay! Feel free to
share this with your friends!
```

File: workshops/personal_website/README.md [hunk 13]
```diff
@@ -0,0 +1,575 @@
...
+Now that you have finished building your website
+
+1. In a new tab, open and follow [these directions][slack] to signup for our
+   Slack
+2. Join your club's channel by asking your club leader for the name of the
+   channel
+3. Post the link to your website
+  - In your club's channel
+4. Post it in the [Ship It](https://shipit.hackclub.io) gallery!
+[slack]: ../../SLACK.md
```
Comment (zachlatta) [inline workshops/personal_website/README.md]: This line should have a newline before it

File: workshops/personal_website/README.md [hunk 14]
```diff
@@ -0,0 +1,575 @@
...
+|-----------------------------------------------------------------------------------|------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------|
+| [HTML&nbsp;Dog](http://www.htmldog.com/guides/html/beginner/)                     | Very beginner focused. If you're not sure which one of these to choose, choose this one. | Isn't too rich in content.                                                      |
+| [Free&nbsp;Code&nbsp;Camp](http://www.freecodecamp.com/map)                       | Interactive and very methodical.                                                         | Not made for you to learn to make something that you want to show your friends. |
+| [Team&nbsp;Tree&nbsp;House](https://teamtreehouse.com/library/html/introduction/) | Their videos are extremely comprehensive an thorough                                     | It takes a _very_ long time to get through and are very passive.                |
```
Comment (zachlatta) [inline workshops/personal_website/README.md]: What is the `&nbsp;` stuff for? Should that just be replaced with spaces?

File: workshops/personal_website/README.md (hunk 5 above)
Comment (zachlatta) [inline workshops/personal_website/README.md]: Yeah, you're right. The entire workshop's header capitalization is
inconsistent. I'll submit a pull request later to make it consistent.

On Fri, Jan 29, 2016 at 12:40 AM, Andrew Downing <notifications@github.com>
wrote:

> In workshops/personal_website/README.md
> <https://github.com/hackclub/hackclub/pull/677#discussion_r51227768>:
>
> > +### 1) Create the HTML file for your website
> > +
> > +1. Right click the **`personal_website`** folder you just created
> > +2. Click **"New File"**
> > +3. Then name the file **`index.html`**
> > +4. Double click it to open it
> > +
> > +> 
> > +
> > +> Notes:
> > +>
> > +> - Just like Microsoft Word Documents end in **`.doc`**, HTML files end in
> > +**`.html**`
> > +> - Your main HTML file has to be named **`index.html`**
> > +
> > +### 2) Adding standard HTML template
>
> In general the workshop is pretty inconsistent in terms of header
> capitalization. I personally don't think it's a big deal. Is this one of
> the things that investors kept pointing out (in addition to spelling
> errors)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/677/files#r51227768>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>

Comment (zachlatta) [inline workshops/personal_website/README.md]: And no, this isn't something that investors are pointing out. Just a pet
peeve for inconsistency.

On Fri, Jan 29, 2016 at 12:43 AM, Zach Latta <zach@zachlatta.com> wrote:

> Yeah, you're right. The entire workshop's header capitalization is
> inconsistent. I'll submit a pull request later to make it consistent.
>
> On Fri, Jan 29, 2016 at 12:40 AM, Andrew Downing <notifications@github.com
> > wrote:
>
>> In workshops/personal_website/README.md
>> <https://github.com/hackclub/hackclub/pull/677#discussion_r51227768>:
>>
>> > +### 1) Create the HTML file for your website
>> > +
>> > +1. Right click the **`personal_website`** folder you just created
>> > +2. Click **"New File"**
>> > +3. Then name the file **`index.html`**
>> > +4. Double click it to open it
>> > +
>> > +> 
>> > +
>> > +> Notes:
>> > +>
>> > +> - Just like Microsoft Word Documents end in **`.doc`**, HTML files end in
>> > +**`.html**`
>> > +> - Your main HTML file has to be named **`index.html`**
>> > +
>> > +### 2) Adding standard HTML template
>>
>> In general the workshop is pretty inconsistent in terms of header
>> capitalization. I personally don't think it's a big deal. Is this one of
>> the things that investors kept pointing out (in addition to spelling
>> errors)?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/hackclub/hackclub/pull/677/files#r51227768>.
>>
>
>
>
> --
> Zach Latta
> https://zachlatta.com <http://zachlatta.com>
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>

**Comment by zachlatta** [comment]
Left a bunch of comments for formatting stuff. Only major thing: sometimes the workshop refers to `Drizzy Drake` and other times it refers to `Drake Graham`. The screenshots seem to say `Drizzy Drake`, so let's change all the content to that.

**Comment by ad510** [comment]
Fixed everything except for the inconsistent header capitalization, which can be fixed in a later PR. Did I get everything, and is there anything else that needs changing?

**Comment by zachlatta** [comment]
All looks good in a preliminary skim. Can you squash these commits?

**Comment by ad510** [comment]
Is it possible to squash while giving @jonleung credit for writing the workshop?

**Comment by zachlatta** [comment]
Yeah, if you squash them it'll say that the commit was authored by @jonleung, but committed by you.

**Comment by zachlatta** [comment]
Sweet! You're the best @ad510 :-).

**Comment by paked** [comment]
Woo!

**Comment by jonleung** [comment]
Thanks so much for all of your help on this!

**Pull request merged**